package.json

```diff
@@ -36,14 +36,14 @@
     "@grpc/proto-loader": "^0.7.13",
     "@penumbra-labs/registry": "^12.0.0",
     "@penumbra-zone/bech32m": "^10.0.0",
-    "@penumbra-zone/client": "^21.0.0",
+    "@penumbra-zone/client": "^21.1.0",
     "@penumbra-zone/crypto-web": "^30.0.0",
     "@penumbra-zone/getters": "^20.0.0",
     "@penumbra-zone/perspective": "^40.0.0",
     "@penumbra-zone/protobuf": "^6.3.0",
     "@penumbra-zone/transport-dom": "^7.5.0",
     "@penumbra-zone/types": "^26.3.0",
-    "@penumbra-zone/ui": "^13.4.2",
+    "@penumbra-zone/ui": "^13.5.0",
```

Comment:
Looks like we have some density issues:

<img width="398" alt="Screenshot 2024-12-11 at 2 28 57 PM" src="https://github.com/user-attachments/assets/0d079f44-ee59-440e-b7f7-1cfb25a88f45" />

Reply:
density updates from the UI package bump are addressed in https://github.com/penumbra-zone/dex-explorer/pull/204 cc @VanishMax